.docs/content/2.usage/use-armonik-api-in-an-angular-app.md

```diff
@@ -0,0 +1,549 @@
+::
+
+## Prerequisites
+
+Before you start, make sure you have the following:
+
+- [Node.js](https://nodejs.org/en/) installed on your machine.
+- [ArmoniK] up and running on your machine. Follow the [installation guide](https://aneoconsulting.github.io/ArmoniK/installation) to install ArmoniK if you haven't done it yet.
+
+::alert{type="info"}
+Please, be aware to install the latest LTS version of Node.js.
+::
+
+## Create a new Angular App
+
+To create a new Angular App, you can use the [Angular CLI](https://angular.io/cli).
+
+```bash
+npm install -g @angular/cli
+```
+
+Then, you can create a new Angular App using the following command:
+
+```bash
+ng new --standalone --routing=false --inline-style --inline-template --skip-tests --style=css --skip-install armonik-api-angular
+```
+
+Then you can go to the newly created folder:
+
+```bash
+cd armonik-api-angular
+```
+
+And install the dependencies:
+
+```bash
+pnpm install
+```
+
+::alert{type="info"}
+We recommend the use of [pnpm](https://pnpm.io/) to install the dependencies. It is faster and more efficient than `npm` or `yarn`.
+::
+
+## Clean the project
+
+In the `src/app/app.component.ts` file, replace the content with the following:
+
+```typescript [app.component.ts]
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-root',
+  template: `
+  `,
+  styles: [`
+  `],
+  standalone: true,
+  imports: [],
+  providers: [],
+})
+export class AppComponent {
+}
+```
+
+## Install ArmoniK API
+
+In order to be able to use ArmoniK API, you need to install the package `@aneoconsultingfr/armonik.api.angular` and some other dependencies.
+
+```bash
+pnpm install @ngx-grpc/common @ngx-grpc/core @ngx-grpc/grpc-web-client google-protobuf @types/google-protobuf @aneoconsultingfr/armonik.api.angular
+```
+
+`google-protobuf` is a dependency that will be used to serialize and deserialize the messages.
+
+::alert{type="info"}
+You can read more about `google-protobuf` in the [official documentation](https://www.npmjs.com/package/google-protobuf).
+::
+
+`@ngx-grpc/common` `@ngx-grpc/core` `@ngx-grpc/grpc-web-client` are dependencies that will allow us to use gRPC in our Angular App.
+
+::alert{type="info"}
+You can read more about `@ngx-grpc` in the [official documentation](https://github.com/ngx-grpc/ngx-grpc).
+::
+
+`@aneoconsultingfr/armonik.api.angular` is the package that contains the ArmoniK API (definition of the gRPC services and the generated code).
+
+::alert{type="info"}
+You can read more about `@aneoconsultingfr/armonik.api.angular` in the [package documentation](https://www.npmjs.com/package/@aneoconsultingfr/armonik.api.angular).
+::
+
+## Configure the Angular App
+
+In order to be able to call some gRPC services, we need to configure the Angular App to use gRPC. In the config file, we will import the providers from the `@ngx-grpc/core` and `@ngx-grpc/grpc-web-client` packages.
+
+In the `src/app/app.config.ts` file, you can add the following content:
```

Comment:
Is it a "you can" as in : 
- "you have the possibility to..." 
or as in 
- "you should add..." ?

Reply:
you should add